ligo/stablecoin/spec.ligo

```diff
@@ -3,7 +3,7 @@
 
 (*
  * Stablecoin types that are specified in
- * https://github.com/tqtezos/stablecoin/blob/317b9346b287a4f2c2606c4929237d87008fdad4/docs/specification.md
+ * https://github.com/tqtezos/stablecoin/blob/master/docs/specification.md
```

Comment:
We usually prefer permanent links because aren't broken if something is moved (e. g. `docs/specification.md` in this case). However, here using `master` looks justified, because permanent link will likely be outdated much faster.